docs/whatsnew/3.0.rst

```diff
@@ -40,6 +40,49 @@ smaller improvements and bug fixes, which are described in the
 Full support for velocities in ``astropy.coordinates``
```

Comment:
maybe this title should be adjusted since the section also mentions the space motion stuff?  Maybe just change "velocities" to "velocities, including space motions" or something like that?

Reply:
Isn't a velocity a space motion?

Reply:
Yes, but I mean we want a title that explicitly makes the point that you can now make space motion *corrections* (or evolutions or whatever you want to call it).  I'm not super attached to the name, though.